### ISSUE TITLE
famous-flex demo doesn't show background on Android Mi 3 phone

### ISSUE THREAD

**Issue by bluemsn** [IjzerenHein/famous-flex#99]
android phone [Mi 3] when i open famous-flex display abnormal ? I don't know why? page show incomplete The  browser  is UC

**Comment by IjzerenHein** [comment]
Do any other famo.us demos work on that phone? What Android version is it running?

**Comment by bluemsn** [comment]
Yes , other browsers display right ~    and [ ios 7, ios 8  ] is ok . 
android   native browser is also ok ~  meybe is  Mi 3' browser UC webview bug??

**Comment by bluemsn** [comment]
I  Send E-mail [ android  phone  screenshots]

**Comment by IjzerenHein** [comment]
Hi, I'm not opening that strange email with a file with no extension. Please post the screenshots here, so everyone can see.

**Comment by bluemsn** [comment]
ok 

![image](https://cloud.githubusercontent.com/assets/3715420/7468165/926f6136-f337-11e4-93d2-2c15cb9f5e58.png)

**Comment by IjzerenHein** [comment]
It looks like that the black background is not shown. I've had a closer look at the demo, and the background was set at a z-index of -10000, and apparently that doesn't work on the browser/GPU of that phone. This was done because of the perspective used for the WheelLayout, which required the background surface to be moved backwards so it doesn't interfere with the translated surfaces of the WheelLayout.

I've updated the demo so that the background surface is now gone, and instead I've made the <body> tag black, in order to achieve the same effect... I've tested it on desktop chrome, iPhone6 and a Galaxy S4 (4.4.2) and it works fine on those.

Let me know whether this fixes the problem?
